python/pyarrow/tests/util.py

```diff
@@ -310,17 +310,7 @@ def open_append_stream(self, path, metadata):
 
 
 def get_raise_signal():
```

Comment:
This means that `get_raise_signal` is not useful anymore, can you remove it and simplify the callers?